README.md

```diff
@@ -63,11 +63,12 @@ Producer
 * `USER_CRT` - the user's certificate
 * `USER_KEY` - the user's private key
 * `LOG_LEVEL` - logging level  
-* `PRODUCER_ACKS` = acknowledgement level
-* `HEADERS` = custom headers list separated by commas of `key1=value1, key2=value2`
-* `ADDITIONAL_CONFIG` = additional configuration for a producer application. The form is `key=value` records separated by new line character
-* `BLOCKING_PRODUCER` = if it's set, the producer will block another message until ack will be received
-* `MESSAGES_PER_TRANSACTION` = how many messages will be part of one transaction. Transaction config could be set via `ADDITIONAL_CONFIG` variable. Default is 10.
+* `PRODUCER_ACKS` - acknowledgement level
+* `HEADERS` - custom headers list separated by commas of `key1=value1, key2=value2`
+* `ADDITIONAL_CONFIG` - additional configuration for a producer application. The form is `key=value` records separated by new line character
```

Comment:
if we are using the ADDITIONAL_CONFIG for even overwriting the ones that can be set via other env vars (i.e. PRODUCER_ACKS) we should say that in the doc at least, otherwise changing the name of the env var to make it clearer but maybe breaking backward compatibility (even if it's just an example so we should be able to do that).